css/builder.css

```diff
@@ -556,6 +556,20 @@ input[type="time"]::-webkit-calendar-picker-indicator {
   opacity: 0;
 }
 
+/* SLIDER FIELD */
+
+[data-widget-package="com.fliplet.form-builder"] .form-group.fl-range-slider.readonly .range-slider-handle-value::before {
+  border-top-color: #e1e1e1;
+}
+
+[data-widget-package="com.fliplet.form-builder"] .form-group.fl-range-slider .range-slider.range-slider-active .range-slider-handle {
+  box-shadow: 0px 0px 0px 2px rgb(0 171 209 / 25%);
+}
+
+.slider-input {
+  width: 100% !important;
```

Comment:
Why is this necessary?

Reply:
Hi @tonytlwu I added this code because while onSlide screen is producing glitch as slider input i having width auto, so its producing scrollbar while doing onslide.

Reply:
<img width="963" alt="image" src="https://user-images.githubusercontent.com/290733/203270598-9a976568-6f52-4f66-a7c4-af3450414127.png">

@NikitaJadhavXoriant See above. [The prototype](https://codepen.io/tonytlwu/pen/QWrzERN) showed that the slider range library should give it styles such that it has 1 x 1 size and is absolutely positioned. What's causing this to behave the way you described and have a different width? If anything, the width override should be `1px`?

And if you really must use a style override, you should use a selector that won't stop other users from using a `slider-input` class, which is pretty generic, e.g. `[data-widget-package="com.fliplet.form-builder"] .form-group.fl-range-slider > input`.

Reply:
Hi @tonytlwu please review this [comment](https://weboo.atlassian.net/browse/XOR-303?focusedCommentId=39050).